runelite-client/src/main/java/net/runelite/client/plugins/party/PartyPlugin.java

```diff
@@ -413,6 +443,15 @@ public void onUserSync(final UserSync event)
 			final SkillUpdate prayUpdate = new SkillUpdate(Skill.PRAYER, currentPrayer, realPrayer);
 			prayUpdate.setMemberId(localMember.getMemberId());
 			ws.send(prayUpdate);
+
+			switch (client.getGameState())
```

Comment:
You can just check localPlayer for being non-null in the method instead of doing game state checks